README.md

```diff
@@ -218,3 +218,12 @@ Enter Spacelift endpoint (eg. https://unicorn.app.spacelift.io/):
 In the next step, you will be asked to choose which authentication method you are going to use. Note that if your account is using [SAML-based SSO authentication](https://docs.spacelift.io/integrations/single-sign-on), then API keys and login with a browser are your only options. After you're done entering credentials, the CLI will validate them against the server, and assuming that they're valid, will persist them in a credentials file in `.spacelift/${MY_ALIAS}`. It will also create a symlink in `${HOME}/.spacelift/current` pointing to the current profile.
 
 You can switch between account profiles by using `spacectl profile select ${MY_ALIAS}`. What this does behind the scenes is point `${HOME}/.spacelift/current` to the new location. You can also delete stored credetials for a given profile by using the `spacectl profile logout ${MY_ALIAS}` command.
+
+## Releasing
+
+To release a new version of spacectl, tag the repo and then push that tag. For example:
```

Comment:
?
```suggestion
To release a new version of spacectl, tag the repo and then push that tag. The build will automatically publish the release. For example:
```

Reply:
Makes sense. I've actually put it under the code example, and just added a link to the workflow as well.

Reply:
idk if github has bugs or something but I can't see your commit in the UI 🤔 weird